internal/cmd/envs.go

```diff
@@ -633,6 +634,5 @@ func autoStartInfo() {
 	} else {
 		preferencesURI = fmt.Sprintf("%s%s", accessURI, "/preferences?tab=autostart")
 	}
-
 	clog.LogInfo("⚡NEW: Automate daily environment startup", "Visit "+preferencesURI+" to configure your preferred time")
```

Comment:
```suggestion
	// clog.LogInfo("⚡NEW: Automate daily environment startup", "Visit "+preferencesURI+" to configure your preferred time")
```

Reply:
This function doesn't get called anymore.